### PR TITLE
Add explicit import of UnrestrictedUser in addition of Super to fix import errors

### PR DESCRIPTION
**Pull request by mpeeters**
See issue #94 and issue #35

### CODE REVIEW

File: src/AccessControl/User.py [hunk 1]
```diff
@@ -35,6 +35,9 @@
 from AccessControl.users import system
 
 
+from AccessControl.users import UnrestrictedUser # noqa isort:skip
```
Comment (gotcha) [inline src/AccessControl/User.py]: What is the reason for `isort:skip` ?

Comment (mpeeters) [reply src/AccessControl/User.py]: To fix isort check and travis build : https://travis-ci.org/zopefoundation/AccessControl/jobs/637872954?utm_medium=notification&utm_source=github_status

Comment (gotcha) [reply src/AccessControl/User.py]: If I understand well, `isort` does not like `#noqa`. I'd add a comment explaining this.

Comment (mpeeters) [reply src/AccessControl/User.py]: isort does not use `noqa` comment, if `isort:skip` is not specified the import is removed by isort.

`noqa` is for Flake8 / Pep8

**Pull request merged**